File: sdkwork-app-sdk-typescript/src/types/search-statistics-vo.ts
/** 搜索统计响应 */
export interface SearchStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 总搜索次数 */
  totalSearches?: number;
  /** 今日搜索次数 */
  todaySearches?: number;
  /** 本周搜索次数 */
  thisWeekSearches?: number;
  /** 本月搜索次数 */
  thisMonthSearches?: number;
  /** 热门搜索词 */
  hotKeywords?: Record<string, number>;
  /** 搜索类型分布 */
  typeDistribution?: Record<string, number>;
  /** 最后搜索时间 */
  lastSearchTime?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/search_statistics_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class SearchStatisticsVO:
    """搜索统计响应"""
    created_at: str = None
    updated_at: str = None
    total_searches: int = None
    today_searches: int = None
    this_week_searches: int = None
    this_month_searches: int = None
    hot_keywords: Dict[str, int] = None
    type_distribution: Dict[str, int] = None
    last_search_time: str = None
